<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ConfigSettingsWidget</class>
 <widget class="QWidget" name="ConfigSettingsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>488</width>
    <height>468</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="settingsTabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_7" columnstretch="1,0">
       <item row="2" column="0">
        <widget class="QCheckBox" name="timelineCreateCheckBox">
         <property name="text">
          <string>Enable hourly snapshots</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="QCheckBox" name="backgruondComparisonCheckBox">
         <property name="text">
          <string>Enable background compare pre and post snapshots after creation</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="permGroupBox">
         <property name="title">
          <string>Users and groups allowed to operate with config</string>
         </property>
         <property name="checkable">
          <bool>false</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Users</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Groups</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0" colspan="2">
           <widget class="CheckableListWidget" name="allowUsersListWidget"/>
          </item>
          <item row="1" column="2">
           <widget class="CheckableListWidget" name="allowGroupsListWidget"/>
          </item>
          <item row="2" column="0" colspan="3">
           <layout class="QHBoxLayout" name="horizontalLayout_4">
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QCheckBox" name="syncAclCheckBox">
              <property name="text">
               <string>Sync users and groups to ACL for snapshot folder</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_4">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Cleanup</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QGroupBox" name="numberCleanupGroupBox">
         <property name="title">
          <string>Enable number cleanup algorithm</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="1" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>How many snapshots should keep</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Minimal age for snapshots to be deleted</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="numberMinAgeSpinBox">
            <property name="maximum">
             <number>2147483647</number>
            </property>
            <property name="value">
             <number>1800</number>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="RangeSpinBox" name="numberLimitSpinBox" native="true"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>How many important snapshots should keep</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="RangeSpinBox" name="numberLimitImportantSpinBox" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="timelineCleanupGroupBox">
         <property name="title">
          <string>Enable timeline cleanup algorithm</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_8">
          <item row="2" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>How many daily snapshots should keep</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QSpinBox" name="timelineMinAgeSpinBox">
            <property name="maximum">
             <number>2147483647</number>
            </property>
            <property name="value">
             <number>1800</number>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="RangeSpinBox" name="timelineLimitMonthlySpinBox" native="true">
            <property name="value" stdset="0">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="RangeSpinBox" name="timelineLimitYearlySpinBox" native="true">
            <property name="value" stdset="0">
             <number>10</number>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Minimal age for snapshots to be deleted</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>How many yearly snapshots should keep</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>How many weekly snapshots should keep</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="RangeSpinBox" name="timelineLimitWeeklySpinBox" native="true"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>How many monthly snapshots should keep</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="RangeSpinBox" name="timelineLimitHourlySpinBox" native="true"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>How many hourly snapshots should keep</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="RangeSpinBox" name="timelineLimitDailySpinBox" native="true"/>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="emptyPrePostClenupGroupBox">
         <property name="title">
          <string>Enable the empty-pre-post cleanup algorithm</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>false</bool>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Minimal age for snapshots to be deleted</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="emptyPrePostMinAgeSpinBox">
            <property name="maximum">
             <number>2147483647</number>
            </property>
            <property name="value">
             <number>1800</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="btrfsTab">
      <attribute name="title">
       <string>btrfs</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="0">
        <widget class="QGroupBox" name="groupBox_6">
         <property name="title">
          <string>Limit for the fraction of the filesystems space the snapshots should use</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QSpinBox" name="spaceLimitSpinBox">
            <property name="suffix">
             <string>%</string>
            </property>
            <property name="maximum">
             <number>100</number>
            </property>
            <property name="value">
             <number>50</number>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>The btrfs quota group</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QComboBox" name="qgroupComboBox">
            <item>
             <property name="text">
              <string>None</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CheckableListWidget</class>
   <extends>QListWidget</extends>
   <header>widgets.CheckableListWidget</header>
  </customwidget>
  <customwidget>
   <class>RangeSpinBox</class>
   <extends>QWidget</extends>
   <header>widgets.RangeSpinBox</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>settingsTabWidget</tabstop>
  <tabstop>permGroupBox</tabstop>
  <tabstop>allowUsersListWidget</tabstop>
  <tabstop>allowGroupsListWidget</tabstop>
  <tabstop>timelineCreateCheckBox</tabstop>
  <tabstop>backgruondComparisonCheckBox</tabstop>
  <tabstop>numberCleanupGroupBox</tabstop>
  <tabstop>numberMinAgeSpinBox</tabstop>
  <tabstop>timelineCleanupGroupBox</tabstop>
  <tabstop>timelineMinAgeSpinBox</tabstop>
  <tabstop>emptyPrePostClenupGroupBox</tabstop>
  <tabstop>emptyPrePostMinAgeSpinBox</tabstop>
  <tabstop>qgroupComboBox</tabstop>
  <tabstop>spaceLimitSpinBox</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
